<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="aspiration">
     <property name="text">
      <string>../data/aspiration.jpg</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="runningBtn">
     <property name="text">
      <string>Прогон</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="initBtn">
     <property name="text">
      <string>Инициализация</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="startBtn">
     <property name="text">
      <string>Старт</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0" colspan="2">
    <widget class="QPushButton" name="clearBtn">
     <property name="text">
      <string>Очистка</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="originBtn">
     <property name="text">
      <string>Оригинал</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="stopBtn">
     <property name="text">
      <string>Стоп</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="origin">
     <property name="text">
      <string>../data/origin.jpg</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QPushButton" name="testBtn_2">
     <property name="text">
      <string>Тестовый</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="test">
     <property name="text">
      <string>../data/test.jpg</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="aspirationBtn">
     <property name="text">
      <string>Преобразованный</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QPushButton" name="requestBtn">
     <property name="text">
      <string>Запрос</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLineEdit" name="result">
     <property name="text">
      <string>../data/result.jpg</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QPushButton" name="testBtn">
     <property name="text">
      <string>Тест</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="18">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
